### PR TITLE
[MIG] project_sequence: Migration to 18.0

### PR DESCRIPTION
**Pull request by BT-mchechlacz**

### CODE REVIEW

**Review by BT-nfrancisco — APPROVED** [review]

**Review by BT-anieto — APPROVED** [review]

File: project_sequence/data/ir_sequence.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Moduon Team S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0) -->
+<data noupdate="1">
```
Comment (BT-ojossen) [inline project_sequence/data/ir_sequence.xml]: [change] use:
odoo noupdate="1"

File: project_sequence/i18n/de.po [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+msgstr ""
+"Verwenden Sie %(sequence_code)s und %(name)s, um den Sequenzcode und den "
+"Namen des Projekts in den Anzeigenamen aufzunehmen."
+
+#~ msgid ""
+#~ "Name of your project. It can be anything you want e.g. the name of a "
+#~ "customer or a service."
+#~ msgstr ""
+#~ "Name Ihres Projekts. Es kann alles sein, was Sie wollen, z.B. der Name "
+#~ "eines Kunden oder einer Dienstleistung."
```
Comment (BT-ojossen) [inline project_sequence/i18n/de.po]: [Question] can this be removed?

File: project_sequence/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+#~ msgstr "Nombre"
+
+#~ msgid "ID"
+#~ msgstr "ID"
+
+#~ msgid "Last Modified on"
+#~ msgstr "Última modificación el"
+
+#~ msgid "Code"
+#~ msgstr "Código"
```
Comment (BT-ojossen) [inline project_sequence/i18n/es.po]: [Question] can this be removed?

File: project_sequence/i18n/it.po [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+#~ msgid ""
+#~ "Name of your project. It can be anything you want e.g. the name of a "
+#~ "customer or a service."
+#~ msgstr ""
+#~ "Nome del progetto. Può essere qualsiasi cosa es. il nome di un cliente o "
+#~ "di un servizio."
+
+#~ msgid "WBS element"
+#~ msgstr "Elemento WBS"
```
Comment (BT-ojossen) [inline project_sequence/i18n/it.po]: [Question] can this be removed?

File: project_sequence/i18n/nl.po [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+msgstr ""
+"Gebruik %(sequence_code)s en %(name)s om de sequentiecode en de naam van het "
+"project op te nemen in de weergavenaam."
+
+#~ msgid ""
+#~ "Name of your project. It can be anything you want e.g. the name of a "
+#~ "customer or a service."
+#~ msgstr ""
+#~ "Naam van uw project. Het kan alles zijn wat je wilt, b.v. de naam van een "
+#~ "klant of dienst."
```
Comment (BT-ojossen) [inline project_sequence/i18n/nl.po]: [Question] can this be removed?

**Review by BT-ojossen — CHANGES_REQUESTED** [review]
LGTM, just some small things to check.

**Review by coleste — APPROVED** [review]
LGTM

**Review by BT-dmoreno — APPROVED** [review]
LGTM

**Review by BT-ojossen — APPROVED** [review]

**Review by dreispt — APPROVED** [review]

**Comment by norlinhenrik** [comment]
Hello! Thank you for migrating this to 18.0. I see that many features in `project_sequence` and `project_task_code` are general. And I have just made an [issue](https://github.com/OCA/server-ux/issues/1058) to discuss two modules for general sequence features. The [repo](https://github.com/loymcom/apps-sequence/tree/18.0) includes [sequence_project](https://github.com/loymcom/apps-sequence/tree/18.0/sequence_project) with sequences for projects and tasks, based on `project_sequence` and `project_task_code`.

Please try it out and let me know what you think. Is it a **good** or **bad** idea to replace `project_sequence` and `project_task_code` with `sequence_project`?

**Pull request merged**